examples/pose-image-filter/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
⚠️ this one is not working at the moment, but it's because the original sandbox got tweaked with new `hoverable` changes but it's not yet working because of `default` keyword usage